src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,16 +56,17 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
+    List<Message> messages = new ArrayList<>();
```

Comment:
```suggestion
    List<Message> messages;
```

Reply:
@Meg1456 Might wanna initialize it like that instead coz IDE flags it as redundancy. (Sorry for missing out this line the first time round and thanks alot for informing on the gc!!!)